### ISSUE TITLE
Issue with the "github-release" concourse resource type tagged versions.

### ISSUE THREAD

**Issue by lucaspinto** [orange-cloudfoundry/cf-ops-automation#220]
First of all, it should be noted that this issue affect the use of github-based BOSH releases defined in `deployment-dependencies.yml` files, used by COA to create BOSH deployments.

### Expected behavior

When using the "github-release" resource type, one has to possibility to chose which version of it should be used via the keyword `tag`. This makes the resource "get" definition look like this:

```yaml
- get: vault
   trigger: true
   attempts: 2
   version:
     tag: v1.0.1
```

When downloaded on the container, the resource will then appear under the name `<releases-name>/release-<release-version>.tgz`, which gives `vault/release-1.0.1.tgz` in our example. Note that the "v" has disappeared.

### Observed behavior

Currently, the version/tag is ignored in the manifest, which prevents us of using this resource type for bosh deployments.

This problem concerns at least the "bosh-geneted" pipeline defined in [`concourse/pipelines/template/bosh-pipeline.yml.erb`](https://github.com/orange-cloudfoundry/cf-ops-automation/blob/develop/concourse/pipelines/template/bosh-pipeline.yml.erb)

### Affected release

Reproducible on version 3.2.2, and probably on sooner version.

### Traces and logs

The task concatenated stdout/stderr looks like this:

```
Invalid release name: vault/release.tgz does not match any files
```

Indeed, on the container, the file is to be found under the tagged name:

```bash
bash-4.4# ls vault
body             commit_sha       tag              vault-1.0.1.tgz  version
```

**Comment by lucaspinto** [comment]
Replacing 

https://github.com/orange-cloudfoundry/cf-ops-automation/blob/84ee7d8ca7b84328ba80f1332efa4650a95151b2/concourse/pipelines/template/bosh-pipeline.yml.erb#L646

with

```yaml
      - <%= release %>/release-((<%= release %>-version)).tgz
```

may suffice to do the trick.

**Comment by o-orand** [comment]
Closed by #221
